assets/styles/button.css

```diff
@@ -30,4 +32,10 @@
   font-size: var(--p);
   border-radius: 16px;
   padding: 12px 32px;
+  font-weight: bold;
+
+}
+.button-danger{
+  background-color: rgb(212, 95, 95);
```

Comment:
I have used hex code to write my color maybe it should be written the same? #f4a3a3